File: java/main/java/src/kata/two/SecondKataTest.java
package kata.two;

import org.testng.annotations.Test;

import static org.testng.Assert.assertEquals;

/**
 * Created by Eureka.
 * User: ggarcia
 * Date: 4/26/11
 * Time: 15:48
 *
 * @author ggarcia
 */
public class SecondKataTest {
    @Test
    public void testSum() {
        SecondKata kata = new SecondKata();
        assertEquals(kata.sum(), 4613732, "4613732");
    }

    @Test
    public void testWithMath() {
        SecondKata kata = new SecondKata();
        assertEquals(kata.sumMath(4000000), 4613732l, "4613732");
        
        assertEquals(kata.sumMath(50), 44, "44");

        assertEquals(kata.sumMath(150), 188, "188");

    }
}

File: ruby/test/02/second_kata_test.rb
require "test/unit"
require "../../katas/02/second_kata"

class SecondKataTest < Test::Unit::TestCase
  def test_sum
    assert_equal(4613732, SecondKata.new.sum(4000000), "result should be 4613732")
	
    assert_equal(44, SecondKata.new.sum(50), "result should be 44")
    
	assert_equal(188, SecondKata.new.sum(150), "result should be 188")
  end
end
